Documentation/cmdref/cilium-agent.md

```diff
@@ -63,6 +63,7 @@ cilium-agent [flags]
       --disable-conntrack                                    Disable connection tracking
       --disable-endpoint-crd                                 Disable use of CiliumEndpoint CRD
       --disable-iptables-feeder-rules strings                Chains to ignore when installing feeder rules.
+      --dns-max-ips-per-restored-rule int                    Maximum number of IPs to maintain for each restored DNS rule (default 1000)
```

Comment:
Shouldn't we alias this to the old arg name at the very least to maintain compatibility? Then in 1.10, we can remove the old arg name.

Reply:
The old name was just introduced and has never been released.

Reply:
Ah, OK I didn't realize. :+1: